conf/application.conf

```diff
@@ -172,6 +172,7 @@ contact-frontend {
 features {
   email-journey = true
   sa = true
+  sia = true
```

Comment:
we should raise an app-config-prod PR as well to disable this flag to make sure we don't accidentally deploy with SIA enabled there